server/api/org/index.ts

```diff
@@ -39,4 +42,10 @@ router.post(
   controller.addOrg,
 );
 
+router.get(
```

Comment:
It seems a bit awkward to have this endpoint here, maybe a new `/api/export/:orgId` endpoint?

Reply:
That's how I originally had it, but it seemed like we might end up with only one route in that endpoint (at least for the foreseeable future). Since it's an export of the org data it felt appropriate that it would go in the org route to me. I'm fine with changing it if you feel strongly about it though.